This file is part of MXE.
See index.html for further information.

Fix compilation of SSE2 sources with GCC 4.9. Since this requires an autoreconf
run, we manually disable the GTK 1.2 dependency, because we don't have the GTK
M4 files available.

diff -Naur lame-3.99.5.orig/configure.in lame-3.99.5/configure.in
--- lame-3.99.5.orig/configure.in	2014-05-02 13:00:26.006106005 +0200
+++ lame-3.99.5/configure.in	2014-05-02 13:14:01.690106046 +0200
@@ -393,7 +393,8 @@
 
 dnl configure use of features
 
-AM_PATH_GTK(1.2.0, HAVE_GTK="yes", HAVE_GTK="no")
+HAVE_GTK="no"
+AC_SUBST(GTK_CFLAGS)
 
 dnl ElectricFence malloc debugging
 AC_MSG_CHECKING(use of ElectricFence malloc debugging)
diff -Naur lame-3.99.5.orig/libmp3lame/vector/Makefile.am lame-3.99.5/libmp3lame/vector/Makefile.am
--- lame-3.99.5.orig/libmp3lame/vector/Makefile.am	2014-05-02 13:00:26.002106005 +0200
+++ lame-3.99.5/libmp3lame/vector/Makefile.am	2014-05-02 13:08:03.854106010 +0200
@@ -20,6 +20,7 @@
 
 if WITH_XMM
 liblamevectorroutines_la_SOURCES = $(xmm_sources)
+liblamevectorroutines_la_CFLAGS = -msse
 endif
 
 noinst_HEADERS = lame_intrin.h
